### PR TITLE
[refactor]Remove sequential file

### PR DESCRIPTION
**Pull request by platoneko**
# Proposed changes

Issue Number: close #xxx

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

This pr is to pave the way for the subsequent IO layer refactoring.

### CODE REVIEW

**Comment by yiguolei** [comment]
LGTM

**Comment by jackwener** [comment]
> This pr is to pave the way for the subsequent IO layer refactoring.

I suggest that we can create a issue to describe the whole picture. 

And split it to many part, we can track this way.

**Review by jackwener — DISMISSED** [review]
LGTM

`sql file` is unused code.

File: be/src/util/storage_backend_mgr.cpp [hunk 1]
```diff
@@ -15,17 +15,17 @@
 // specific language governing permissions and limitations
 // under the License.
```
Comment (yiguolei) [inline be/src/util/storage_backend_mgr.cpp:16]: Do not refactor storage_backend_mgr.cpp or .h， focus on refactor sequence file， not others.

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**